sebastien15/capstoneproject
# Ft retrieve comments

#### What does this PR do?

> this PR retrieve all comments from the database and display the results

#### Description of Task to be completed
> Added files commentController, Comment and commentRouter
> Added method to retrieve comment in comment Controller
>added comment route to app.js

#### How should this be manually tested?

- Clone this repo with `git clone git@github.com:sebastien15/myportifolio.git` 
- Checkout to `ft-retrieve-comments`
- npm install nodemon -g
- npm install
- npm start
- run localhost:3000/allComments in post man make sure it is a GET method


#### Any background context you want to provide?
>N/A
#### What are the relevant Trello story?
>https://trello.com/c/JblBHkE9
#### Screenshots (if appropriate)
>
![retreive comment](https://user-images.githubusercontent.com/41580903/94120933-ff29f500-fe50-11ea-91e2-fcb3aa62bc94.PNG)
